rosidl_runtime_c/include/rosidl_runtime_c/service_type_support_struct.h

```diff
@@ -81,12 +82,18 @@ struct rosidl_service_type_support_t
   const void * data;
   /// Pointer to the service type support handler function
   rosidl_service_typesupport_handle_function func;
+  /// Service request message typesupport
+  const rosidl_message_type_support_t * request_typesupport;
+  /// Service response message typesupport
+  const rosidl_message_type_support_t * response_typesupport;
```

Comment:
Yes, I debated about this. I'd like to keep the ordering of "request, response, event", but I also want to keep all the "event" stuff together. Which is why I did this as

```
request_typesupport
response_typesupport
event_message_create_function
event_message_destroy_function
event_typesupport
```

To keep both groups together, it could be instead

```
request_typesupport
response_typesupport
event_typesupport
event_message_create_function
event_message_destroy_function
```

Reply:
Yeah, I like that.

Reply:
Done